portal/common/lib/rpc_selector.ts

```diff
@@ -44,12 +43,25 @@ class RPCSelector implements RPCSelectorInterface {
         this.selectedClient = undefined;
     }
 
+    /**
+     * Get the list of RPC URLs from the environment variable.
+     */
+    private static parseRPCList(): string[] {
+        const rpcList = process.env.RPC_URL_LIST;
```

Comment:
Question: does this work even when running the portal locally?

Reply:
Yes, as long as the environment variable is specified. We might need to provide a `.env.example` to our repo for people that want to do zero configuration.